bugbug/repository.py

```diff
@@ -114,6 +125,12 @@ def download_commits(repo_dir):
     commits = hg_log(repo_dir)
     commits_num = len(commits)
 
+    subprocess.run([os.path.join(repo_dir, './mach'), 'file-info', 'bugzilla-automation', 'component_data'], cwd=repo_dir)
```

Comment:
Either use subprocess.check_call, or add the check=True parameter. So if the execution fails this will throw an exception.